### ISSUE TITLE
Output scrapped data directly to database.

### ISSUE THREAD

**Issue by mkao006** [EST-Team-Adam/TheReadingMachine#56]
This will reduce the redudant `jsonl` files, which can result in out of memory problem in deployment.

**Comment by mrpozzi** [comment]
Hitting the db multiple time seems like an unnecessary inefficiency (specially as we will be having less links to store locally). Let's minimize communication since that could be a bottleneck harder to solve than increasing memory on the instance.

**Comment by mkao006** [comment]
I don't think this is a significant overhead if you only hit the database once each source is completed. Given there are only 2 functioning sources, that's 2 connections.

As per current implementation, the instance will be overloaded and crash in about 10 days due to insufficient data storage (30GB allocated).

Whether writing straight to database is a good solution, this issue has to be resolved.
